src/hooks/useActiveDescendantListFocus/__stories__/useActiveDescendantListFocus.stories.mdx

```diff
@@ -96,9 +96,47 @@ import styles from "./useActiveDescendantListFocus.module.scss";
   </Story>
 </Canvas>
 
-## Props
+## Arguments
 
-<ArgsTable story="Overview" />
+<FunctionArguments>
```

Comment:
Why are we using this instead? Just curious

Reply:
That's our convention for hooks args documentation
Not 100% sure why, but probably hooks arguments weren't properly documented automatically by ArgsTable...